typescript/infra/config/environments/mainnet3/tokenPrices.json

```diff
@@ -1,112 +1,111 @@
 {
-  "ancient8": "3967.88",
-  "alephzeroevmmainnet": "0.574132",
-  "apechain": "1.55",
-  "appchain": "3967.88",
-  "arbitrum": "3967.88",
-  "arbitrumnova": "3967.88",
-  "arthera": "0.130115",
-  "astar": "0.073381",
-  "astarzkevm": "3967.88",
-  "aurora": "3967.88",
-  "flame": "6.58",
-  "avalanche": "49.43",
-  "b3": "3967.88",
-  "base": "3967.88",
-  "bitlayer": "106640",
-  "blast": "3967.88",
-  "bob": "3967.88",
-  "boba": "3967.88",
-  "bsc": "719.63",
-  "bsquared": "106640",
-  "celo": "0.810897",
-  "cheesechain": "0.00135152",
-  "chilizmainnet": "0.108762",
-  "conflux": "0.205619",
-  "conwai": "0.00468951",
-  "coredao": "1.35",
-  "corn": "106640",
-  "cyber": "3967.88",
-  "degenchain": "0.0142007",
-  "dogechain": "0.401903",
-  "duckchain": "6.06",
-  "eclipsemainnet": "3967.88",
-  "endurance": "2.71",
-  "ethereum": "3967.88",
-  "everclear": "3967.88",
-  "evmos": "0.03072161",
-  "fantom": "1.38",
-  "flare": "0.0303841",
-  "flowmainnet": "0.931723",
-  "form": "3967.88",
-  "fractal": "0.00058043",
-  "fraxtal": "3939.37",
-  "fusemainnet": "0.03904172",
-  "gnosis": "1.001",
-  "gravity": "0.0340731",
-  "harmony": "0.03279832",
-  "immutablezkevmmainnet": "1.8",
-  "inevm": "28.72",
-  "ink": "3967.88",
-  "injective": "28.72",
-  "kaia": "0.275014",
-  "kroma": "3967.88",
-  "linea": "3967.88",
-  "lisk": "3967.88",
-  "lukso": "2.9",
-  "lumia": "2.03",
-  "lumiaprism": "2.03",
-  "mantapacific": "3967.88",
-  "mantle": "1.21",
-  "merlin": "106736",
-  "metal": "3967.88",
-  "metis": "55.88",
-  "mint": "3967.88",
-  "mode": "3967.88",
-  "molten": "0.419836",
-  "moonbeam": "0.309739",
-  "morph": "3967.88",
-  "neutron": "0.532415",
-  "oortmainnet": "0.190901",
-  "optimism": "3967.88",
-  "orderly": "3967.88",
-  "osmosis": "0.58828",
-  "polygon": "0.600874",
-  "polygonzkevm": "3967.88",
-  "polynomialfi": "3967.88",
-  "prom": "6.7",
-  "proofofplay": "3967.88",
-  "rarichain": "3967.88",
+  "ancient8": "3862.21",
```

Comment:
this might be annoying to move from JSON to js so feel free to ignore, but if we use the Eth price for all eth rollups that could just be a variable which would make this easier to review 😄

Reply:
i hear you - but also this format is easiest for just promise.all'ing the tokenprice based on the gecko ID in the registry metadata